selenium-system-test/src/test/scala/hts/pages/Page.scala

```diff
@@ -39,7 +39,7 @@ trait Page extends Matchers
 
   def checkHeader(heading: String, text: String)(implicit driver: WebDriver): Boolean =
     find(cssSelector(heading)).exists(_.text === text)
-currentUrl
```

Comment:
Andy has remove this already in his PR

Reply:
yeah, I accidently put this line in in a previous PR but I corrected it